db/models.py

```diff
@@ -1 +1,33 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race,
+                             on_delete=models.CASCADE,
+                             related_name="skill_set")
```

Comment:
why did you provide a default value for `related_name` arg ?

Reply:
Firstly, I gave it a `skill` value, but one test not passed and i changed `related_name`,not deleted